server/db.js

```diff
@@ -593,3 +593,15 @@ exports.updateQuestionById = function(questionId, request, callback){
         return callback(null, 'success');
     });
 }
+
+exports.removeQuestion = function(questionId, callback){
```

Comment:
Nice work :)
I see you got the hang of editing the backend so quickly ;)
We gotta make some changes to the way we store students' answers. Part of deleting the question is getting rid of its score. The issue that we cant just simply remove the score of the question, because the way we handle score various from one student to another. We have to store the individual points first so we can later remove them. The problem with that is keeping unique entries in the backend, which I am looking into a proper way of handle that without running into race conditions.
Overall, nice work here, but sadly we have to make other changes before this one is in.

Reply:
I agree with Ahmed, I'm pretty sure we haven't figured out how to recalcualte the scores that will be removed once a question is deleted. Also, have we 100% decided that we are removing the points if a question is deleted or are we treating it on a case by cases basis? Aside from that great work.